go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	filippo.io/age v1.1.1
 	github.com/99designs/keyring v1.2.1
 	github.com/armon/go-metrics v0.4.1
-	github.com/cometbft/cometbft v0.37.7
+	github.com/cometbft/cometbft v0.37.9-0.20240703112902-3f438815c797
```

Comment:
https://github.com/cometbft/cometbft/compare/v0.37.7...3f438815c797 do we also align version and store?

Reply:
it's fine I think